workflows/argo-events/workflowtemplates/enroll-server.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: argoproj.io/v1alpha1
+metadata:
+  name: enroll-server
+  annotations:
+    workflows.argoproj.io/title: Perform server discovery and update Nautobot and Ironic
+    workflows.argoproj.io/description: |
+      Defined in `workflows/argo-events/workflowtemplates/enroll-server.yaml`
+kind: WorkflowTemplate
+spec:
+  arguments:
+    parameters:
+      - name: ip_address
+  templates:
+    - name: enroll-server
+      inputs:
+        parameters:
+          - name: ip_address
+      container:
+        image: ghcr.io/rackerlabs/understack/ironic-nautobot-client:pr-370
+        command:
+          - enroll-server
+        args:
+          - --bmc-ip-address
+          - "{{workflow.parameters.ip_address}}"
+        volumeMounts:
+          - mountPath: /etc/openstack
+            name: openstack-svc-acct
+            readOnly: true
+          - mountPath: /etc/nb-token/
+            name: nb-token
+            readOnly: true
+        env:
+        - name: BMC_MASTER
+          valueFrom:
+            secretKeyRef:
+              name: bmc-master
+              key: key
```

Comment:
this one should probably be mounted as volume if possible

Reply:
fixed